test/contracts/token/arcxRedemption.test.ts

```diff
@@ -0,0 +1,101 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { TestToken, TestTokenFactory } from '@src/typings';
+import { ArcxRedemption } from '@src/typings/ArcxRedemption';
+import { MockArcxRedemptionFactory } from '@src/typings/MockArcxRedemptionFactory';
+import { addSnapshotBeforeRestoreAfterEach } from '@test/helpers/testingUtils';
+import { expect } from 'chai';
+import { BigNumber, utils } from 'ethers';
+import { ethers } from 'hardhat';
+
+const ARCX_AMOUNT = BigNumber.from(10).pow(18).mul(100)
+const USDC_AMOUNT = BigNumber.from(10).pow(6).mul(55)
+
+describe('ArcxRedemption', () => {
+  let owner: SignerWithAddress;
+  let user: SignerWithAddress;
+
+  let redemption: ArcxRedemption;
+
+  let exchangeRate: BigNumber;
+  let cutoffDate: BigNumber;
+
+  let usdcToken: TestToken;
+  let arcxToken: TestToken;
+
+  before(async () => {
+    const signers = await ethers.getSigners();
+    owner = signers[0];
+    user = signers[1];
+
+    cutoffDate = BigNumber.from(10);
+    exchangeRate = utils.parseEther('0.5');
+
+    usdcToken = await new TestTokenFactory(owner).deploy("TES", "TEST", 6);
+    arcxToken = await new TestTokenFactory(owner).deploy("TEST", "TEST", 18);
```

Comment:
```suggestion
    usdcToken = await new TestTokenFactory(owner).deploy("USDC", "USDC", 6);
    arcxToken = await new TestTokenFactory(owner).deploy("ARCX", "ARCX", 18);
```
To help for debugging eventually